spec/std/system/group_spec.cr

```diff
@@ -0,0 +1,76 @@
+require "spec"
+require "system/group"
+require "system/user"
+
+module Spec::Group
```

Comment:
Is there a specific reason for this module? You could just put the constants in the top level namespace and make them `private`.